crates/runtime/src/datafusion.rs

```diff
@@ -289,6 +290,7 @@ impl DataFusion {
                     .context(UnableToCreateMemoryStreamSnafu)?,
                 ));
             }
+            metrics::counter!("results_cache_miss").increment(1);
```

Comment:
Do we really need a miss statistic?  We have the request count already, right?

Reply:
Updated